BD-3449 Update UI and screenshots for Billing (#8044)
braze-inc/braze-docs: _docs/_user_guide/administrative/app_settings/subscription_and_usage.md

# Billing

> > Use the **Billing** page as a self-serve tool to monitor and check your data consumption across workspaces, apps, and event sources. This article covers the different sections on the page and the information they can provide you.

Reviewer: ```suggestion
> Learn how to use the **Billing** page to monitor and check your data consumption across workspaces, apps, and event sources. This article covers the different sections on the page and the information they can provide you.
```

## Most used events and attributes by app

> The **Most Used Events and Attributes By App** tab is a useful tool to understand the drivers of your attribute and custom event data point consumption. 

Reviewer: ```suggestion
Under **Most Used Events and Attributes By App**,  you can check the drivers of your attribute and custom event data point consumption. 
```
